CoffeeChat/Controller/HomeViewController.swift

```diff
@@ -68,6 +69,7 @@ class HomeViewController: UIViewController {
         tabCollectionView.layer.shadowRadius = 4
         view.addSubview(tabCollectionView)
 
+        setUpLocalNotifications()
```

Comment:
I think we capitalize it like `setupLocalNotifications` in some other files. We should keep it consistent.